course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -255,6 +342,28 @@
         $scope.renderName = function(data, type, full, meta) {
             return full.profile.name_last + ', ' + full.profile.name_first;
         };
+        $scope.renderRemove = function(data, type, full, meta) {
+            // TODO - maybe make this a directive?  the isolate scope
+            //        in a directive complicates things.  anything has
+            //        to be better than this mess, though.
```

Comment:
I like the approach you outlined above for cleaning this up...makes it much more readable.

The row data is being pulled by datatables as needed...not sure there's good hooks for adding to the prototype there.  And this is called as a peer to the `renderSource()` method, so I don't think we can safely assume that the rendered cell is going to be available for us to inspect.  Anything I come up with to avoid running that regex twice is more complicated, and way less readable, than just running it twice.